src/python/pants/notes/1.26.x.rst

```diff
@@ -3,7 +3,16 @@
 
 This document describes releases leading up to the ``1.26.x`` ``stable`` series.
 
-The ``1.26.x`` series adds support for TOML config files, e.g. ``pants.toml``. INI config files will still be supported for some time, but INI is now legacy. Please see https://groups.google.com/forum/#!topic/pants-devel/N1H03oJONco for more information.
+Some significant changes in this series:
+
+* Pants now uses PEX ``2.1.x``, which uses PIP under the hood, and consequently significantly changes resolves (in order to better align with the wider Python ecosystem). Please see https://github.com/pantsbuild/pex/pull/788 for details.
+
+* Adds support for TOML config files, e.g. ``pants.toml``. INI config files will still be supported for some time, but INI is now legacy. Please see https://groups.google.com/forum/#!topic/pants-devel/N1H03oJONco for more information.
+
+1.26.0 (04/29/2020)
+-------------------
+
+The first stable release in the ``1.26.x`` series, with no changes since the previous rc!
```

Comment:
It'd be good to include https://github.com/pantsbuild/pants/pull/9589 if you don't mind cherry-picking it in.

Reply:
As our process stands now, that would be another rc and another 5 day cooldown, so I think it's probably missed the boat.